src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -87,14 +80,49 @@ export class SkyTabsetComponent
     return this._permalinkId || '';
   }
 
+  /**
+   * @deprecated
+   * Specifies the behavior for a series of tabs.
+   * The property was designed to create wizards by setting tabStyle="wizard" on tabsets in modals,
+   * but this wizard implementation was replaced by the progress indicator component.
```

Comment:
Can we make "the progress indicator component" a link to https://developer.blackbaud.com/skyux/components/progress-indicator?

Reply:
Done.